CI/travis/before_deploy

```diff
@@ -18,8 +18,41 @@ else
 fi
 pwd
 
+if [ -z "${LDIST}" -a -f "build/.LDIST" ] ; then
+	LDIST=$(cat build/.LDIST)
+fi
 if [ -z "${LDIST}" ] ; then
-	LDIST=-$(lsb_release -c | awk '{print $NF}')
+	# provide something like "-Debian-buster-amd64" or
+	# "-Ubuntu-xenial-amd64" or "-CentOS-7.5-x86_64"
+	case "$(uname)" in
```

Comment:
while i like the idea, i think something a bit more elegant would work for Linux ;
maybe, break it down into a separate function (or functions) ; something like:
```
get_ldist() {
   if [ ! -f /etc/os-release ] ; then
        return 1
   fi
   source /etc/os-release
   [ -z "$UBUNTU_CODENAME" ] || {
         echo "$UBUNTU_CODENAME"
         return
   }

    [ -z "REDHAT_SUPPORT_PRODUCT" ] || {
         echo "${REDHAT_SUPPORT_PRODUCT}${REDHAT_SUPPORT_PRODUCT_VERSION}"
          return 0
    }
    return 1
}

LDIST=$(get_ldist)
```

some notes:
* this is untested but overall it should work
* for CentOS, there are other variables to choose from, in order to build the LDIST variable ( e.g. ID for `centos`, VERSION_ID or CENTOS_MANTISBT_PROJECT_VERSION for CentOS version ) ; I chose the REDHAT_ ones because they seemed handy
* the `ID_LIKE` variable can be used to organize distros together ; 
  * for example `ID_LIKE=debian` is common for Ubuntu & Raspbian  ; Debian doesn't have any ID_LIKE, it just has `ID=debian`
  * for CentOS, ID_LIKE seems to be `ID_LIKE="rhel fedora` ; I'd assume that ID_LIKE is a bit different for Fedora & Redhat, but it could still be useful

Reply:
This is a better idea - and I refactored things like that, but didn't use the vendor specific names, sticking with ones in the standard http://0pointer.de/blog/projects/os-release.html